src/django/pfb_network_connectivity/settings.py

```diff
@@ -196,7 +196,7 @@
         },
     },
     'loggers': {
-        'django': {
```

Comment:
The configuration for the `django` logger should be left in, as that captures all messages from the `django` namespace. It makes sense for the `pfb_analysis` logger to duplicate the configuration of the `django` logger